scripts/perftune.py

```diff
@@ -1211,12 +1211,12 @@ def names():
 
 Default values:
 
- --nic NIC       - default: eth0
+ --nic NIC       - default: eth0 (you can specify multiple nics)
  --cpu-mask MASK - default: all available cores mask
  --tune-clock    - default: false
 ''')
 argp.add_argument('--mode', choices=PerfTunerBase.SupportedModes.names(), help='configuration mode')
-argp.add_argument('--nic', help='network interface name, by default uses \'eth0\'')
```

Comment:
Please, change the args.nic to args.nics - it will make it more readable. Keep the `--nic` however: there is a way to define the destination field in the `args`. See how I did it for `--dir`